example/App.tsx

```diff
@@ -86,6 +101,32 @@ export default function App() {
       >
         Success Toast
       </Text>
+      <View style={{ height: 16 }} />
+      <Text
+        style={[styles.text, styles.blue]}
```

Comment:
I don't know which color is better for these buttons, I don't want broking UI so I just copy the above, maybe need you to change it! 😃